tests/check_consistent.py

```diff
@@ -3,9 +3,8 @@
 # Symlinks are bad on Windows, so we cannot use them in typeshed.
 # This checks that certain files are duplicated exactly.
 
-import os
+import os.path
```

Comment:
Even this is not necessary. `import os` guarantees that the `os.path` module is also imported.